### PR TITLE
Create dependabot.yml

### PR DESCRIPTION
**Pull request by lizthegrey**
Adopts GitHub Dependabot for updating dependencies and staying on top of routine & security updates.

### CODE REVIEW

**Comment by lizthegrey** [comment]
Expecting to see a @dependabot GitHub Check in the list as well...

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  - package-ecosystem: "gomod" # See documentation for possible values
+    directory: "/exporters/otlp" # Location of package manifests
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "gomod" # See documentation for possible values
+    directory: "/exporters/metric/prometheus" # Location of package manifests
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "gomod" # See documentation for possible values
+    directory: "/trace/zipkin" # Location of package manifests
```
Comment (Aneurysm9) [inline .github/dependabot.yml:25]:
```suggestion
    directory: "/exporters/trace/zipkin" # Location of package manifests
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "gomod" # See documentation for possible values
+    directory: "/trace/jaeger" # Location of package manifests
```
Comment (Aneurysm9) [inline .github/dependabot.yml:29]:
```suggestion
    directory: "/exporters/trace/jaeger" # Location of package manifests
```

**Comment by lizthegrey** [comment]
Whoops, thanks for the catch.

**Comment by lizthegrey** [comment]
https://github.com/open-telemetry/opentelemetry-go/pull/814/checks?check_run_id=774368149

there's the check.

**Comment by lizthegrey** [comment]
> Is there a reason we didn't include the examples in this?

None in particular, I just felt like it would add a lot of spam. But I'm happy to add.

**Comment by MrAlias** [comment]
> > Is there a reason we didn't include the examples in this?
> 
> None in particular, I just felt like it would add a lot of spam. But I'm happy to add.

Good point. Maybe just start with this :+1:

**Comment by lizthegrey** [comment]
I'm working on the CLA exception now.

**Comment by lizthegrey** [comment]
bot is now CLA authorized.

**Pull request merged**